quinn-proto/src/connection/mod.rs

```diff
@@ -1428,6 +1480,16 @@ impl Connection {
                 );
             }
         }
+
+        // Handle a lost MTU probe
+        if let Some(packet) = lost_mtu_probe {
+            let info = self.spaces[SpaceId::Data]
+                .sent_packets
+                .remove(&packet)
+                .unwrap(); // safe: lost_packets is populated just above
```

Comment:
It's not immediately obvious what lost_packets being populated has to do with this unwrap. Should this say something like "lost_mtu_probe is necessarily omitted from lost_packets, and therefore must not have been removed yet"?

Reply:
Good catch, thanks!